ui/pages/Project/components/RnaSeqOutliers.jsx

```diff
@@ -97,36 +96,30 @@ class RnaSeqOutliersGraph extends React.PureComponent {
 
 }
 
-const BaseRnaSeqOutliers = React.memo(({ individualGuid, rnaSeqData, genesById, familyGuid, loading, load }) => (
-  <DataLoader content={rnaSeqData} contentId={individualGuid} load={load} loading={loading}>
-    <GeneSearchLink
-      buttonText="Search for variants in outlier genes"
-      icon="search"
-      location={Object.values(rnaSeqData || {}).filter(({ isSignificant }) => isSignificant).map(({ geneId }) => geneId).join(',')}
-      familyGuid={familyGuid}
-      floated="right"
-    />
-    <RnaSeqOutliersGraph data={rnaSeqData} genesById={genesById} />
-  </DataLoader>
-))
-
-BaseRnaSeqOutliers.propTypes = {
-  individualGuid: PropTypes.string.isRequired,
+const RnaSeqOutliers = React.memo(({ rnaSeqData, genesById, familyGuid, getLocation, header }) => {
+  const { sampleId, tissueType } = Object.values(rnaSeqData)[0]
+  const data = rnaSeqData
+  return (
+    <div>
+      <Header content={`${header}: ${sampleId}(${TISSUE_DISPLAY[tissueType]})`} textAlign="center" />
```

Comment:
The page header will tell you what sample you are looking at, theres no need to include the  `sampleId`